cmd/auth-handler.go

```diff
@@ -153,7 +154,8 @@ func checkRequestAuthType(ctx context.Context, r *http.Request, action policy.Ac
 	var locationConstraint string
 	if action == policy.CreateBucketAction {
 		// To extract region from XML in request body, get copy of request body.
-		payload, err := ioutil.ReadAll(r.Body)
+		const maxBodySize = 3 * 1024 * 1024 // Limit the body to at most 3 MB (about the same as AWS)
```

Comment:
This can be a global constant and perhaps which indicates that it is a make bucket body maximum? - Perhaps this value can be as big bucket policy ~ PUT operation itself.  I don't see any reason why it needs to be 3MiB.

Reply:
3 MB is round about the size AWS accepts for the create bucket body.
But I think it's okay to use `maxBucketPolicySize` also...
Just didn't made it a global constant since it's not valuable anywhere else and therefore needn't to be in global scope...

Will change to `maxBucketPolicySize`